### PR TITLE
ref(grouping-tab): Add hierarchical grouping new api UI

### PR DESCRIPTION
**Pull request by priscilawebdev**
Adds  hierarchical grouping new api UI in the UI + Introduces a new UI Prototype

**Preview:**

image

### CODE REVIEW

File: static/app/views/organizationGroupDetails/grouping/grouping.tsx [hunk 1]
```diff
@@ -1,139 +1,304 @@
...
-        location={location}
-        issues={mergedIssuesWithLastestEvent}
-        selectedIssueIds={selectedIssueIds}
-        pagination={mergedIssuesPagination}
-        enableCompareButton={enableCompareButton}
-      />
-      <GroupedIssues issues={mergedIssuesWithLastestEvent} groupId={groupId} />
+      <Description>
+        {t(
+          'Sometimes you might want to split up the errors in an issue by different frames in the stacktrace. Below you can select which frames to regroup this issue by and see how many new issues will be created in the process.'
```
Comment (untitaker) [inline static/app/views/organizationGroupDetails/grouping/grouping.tsx:179]:
```suggestion
          'Sometimes you might want to split up issues by additional frames or other criteria. Select a granularity level below and see how many new issues will be created in the process.'
```

File: static/app/views/organizationGroupDetails/grouping/grouping.tsx [hunk 2]
```diff
@@ -1,139 +1,304 @@
...
+            {t('Select levels')}
+            <StyledRangeSlider
+              name="grouping-level"
+              allowedValues={groupingLevels.map(groupingLevel =>
+                Number(groupingLevel.id)
+              )}
+              formatLabel={value => {
+                return value === 0
+                  ? t('Automatically grouped')
+                  : tn('%s level', '%s levels', value);
```
Comment (untitaker) [inline static/app/views/organizationGroupDetails/grouping/grouping.tsx:194]:
```suggestion
                  : t('Level %s', value);
```

File: static/app/views/organizationGroupDetails/grouping/grouping.tsx [hunk 3]
```diff
@@ -1,139 +1,304 @@
...
-      <GroupedIssues issues={mergedIssuesWithLastestEvent} groupId={groupId} />
+      <Description>
+        {t(
+          'Sometimes you might want to split up the errors in an issue by different frames in the stacktrace. Below you can select which frames to regroup this issue by and see how many new issues will be created in the process.'
+        )}
+      </Description>
+      <div>
+        <StyledList symbol="colored-numeric">
+          <StyledListItem>
+            {t('Select levels')}
```
Comment (untitaker) [inline static/app/views/organizationGroupDetails/grouping/grouping.tsx:185]:
```suggestion
            {t('Select level')}
```

File: static/app/views/organizationGroupDetails/grouping/grouping.tsx [hunk 4]
```diff
@@ -1,139 +1,304 @@
...
+                <div>{t('What happens to this issue')}</div>
+                <LoadingIndicator mini />
+              </div>
+            ) : (
+              <Fragment>
+                <div>
+                  {t('What happens to this issue')}
+                  <WhatHappensDescription>
+                    {tct(
+                      `This issue will be deleted and [quantity] new issues will be created.`,
```
Comment (untitaker) [inline static/app/views/organizationGroupDetails/grouping/grouping.tsx:214]:
```suggestion
                      `This issue will be deleted and [quantity]+ new issues will be created.`,
```

Not sure if there's a better solution at the moment, I have created an asana task for providing you with a better API for this number.

**Review by untitaker — APPROVED** [review]

File: static/app/views/organizationGroupDetails/grouping/grouping.tsx [hunk 5]
```diff
@@ -1,139 +1,308 @@
...
 }
 
-export default Grouping;
+export default withApi(Grouping);
 
 const Wrapper = styled('div')`
+  flex: 1;
   display: grid;
-  grid-gap: ${space(1)};
+  background: ${p => p.theme.white};
```
Comment (matejminar) [inline static/app/views/organizationGroupDetails/grouping/grouping.tsx:258]: How does this play with darkmode?

**Review by matejminar — APPROVED** [review]
lgtm

File: static/app/views/organizationGroupDetails/grouping/grouping.tsx (hunk 5 above)
Comment (priscilawebdev) [reply static/app/views/organizationGroupDetails/grouping/grouping.tsx]: thanks.. I've update it 😉

**Pull request merged**